src/activities/UseFmeService.ts

```diff
@@ -16,23 +16,214 @@ export interface UseFmeServiceInputs {
+        | "projects/projects/{project}/export/resource"
+        | "projects/projects/{project}/icon"
+        | "projects/projects/{project}/items"
+        | "projects/projects/{project}/items/resourcepaths/{resource}/{path}"
+        | "projects/projects/{project}/items/{type}/{item}"
+        | "projects/projects/{project}/items/{type}/{scope}/{item}"
+        | "projects/test/projects"
         | "repositories"
+        | "repositories/{repository}"
+        | "repositories/{repository}/items"
+        | "repositories/{repository}/items/{item}"
+        | "repositories/{repository}/items/{item}/datasets/{dtype}"
+        | "repositories/{repository}/items/{item}/datasets/{dtype}/{dataset}"
+        | "repositories/{repository}/items/{item}/datasets/{dtype}/{dataset}/featuretypes"
+        | "repositories/{repository}/items/{item}/datasets/{dtype}/{dataset}/featuretypes/{feature}"
+        | "repositories/{repository}/items/{item}/datasets/{dtype}/{dataset}/featuretypes/{feature}/attributes"
+        | "repositories/{repository}/items/{item}/datasets/{dtype}/{dataset}/featuretypes/{feature}/attributes/{attribute}"
+        | "repositories/{repository}/items/{item}/detailedservices"
+        | "repositories/{repository}/items/{item}/metrics"
+        | "repositories/{repository}/items/{item}/parameters"
+        | "repositories/{repository}/items/{item}/parameters/{pubparam}"
+        | "repositories/{repository}/items/{item}/resources"
+        | "repositories/{repository}/items/{item}/resources/{resource}"
+        | "repositories/{repository}/items/{item}/services"
+        | "repositories/{repository}/items/{item}/services/{service}"
         | "resources/connections"
+        | "resources/connections/{resource}"
+        | "resources/connections/{resource}/copy/{path}"
+        | "resources/connections/{resource}/directdownloadurl/{path}"
+        | "resources/connections/{resource}/downloadzip/{directory}"
+        | "resources/connections/{resource}/download/{path}"
+        | "resources/connections/{resource}/filesys/{path}"
+        | "resources/connections/{resource}/move/{path}"
+        | "resources/connections/{resource}/directdownloadurlparameters"
+        | "resources/connections/{resource}/directuploadurlparameters"
+        | "resources/test/connections"
+        | "resources/test/connections/{resource}"
         | "resources/types"
         | "schedules"
         | "schedules/categories"
+        | "schedules/{category}"
+        | "schedules/{category}/{name}"
+        | "schedules/{category}/{name}/metrics"
+        | "schedules/{category}/{name}/trigger"
+        | "security/accounts"
+        | "security/accounts"
+        | "security/accounts/{account}"
+        | "security/accounts/{account}/email"
+        | "security/accounts/{account}/enabled"
+        | "security/accounts/{account}/ldap"
+        | "security/accounts/{account}/password"
+        | "security/accounts/{account}/password"
```

Comment:
Duplicate entry.

Reply:
I assume this one was meant to be `security/accounts/{account}/passwordChangeNeeded`, which is missing.

Reply:
Fixed